easybuild/easyconfigs/a/atat/atat-3.36-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,44 @@
+# Ruoshi Sun <rs7wz@virginia.edu>
+# Research Computing, University of Virginia
+# 2020-05-23
+
+easyblock = 'MakeCp'
+
+name = 'atat'
```

Comment:
@rsdmse We usually try to stick to the official software name, which would be `ATAT` in this case, do you mind changing that?

Note that this also implies changing the easyconfig file name to `ATAT-3.36-GCCcore-9.3.0.eb`, and the subdirectory it's located in to `a/ATAT`.

You can use `--update-pr` for this, where you can [prefix the name of an easyconfig file with `:` to indicate it should be deleted](https://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html#deleting-easyconfig-files-or-patches).

So, if `ATAT.eb` is the local fixed easyconfig that uses `'ATAT'` as software name, this should work:

```
eb --update-pr 10674 :atat-3.36-GCCcore-9.3.0.eb ATAT.eb
```

You can use `eb --dry-run --update-pr 10674 ...` to verify this before taking any action.

Reply:
@boegel Thanks for the tips. I changed the name to ATAT. The `BINDIR` variable is only invoked during `make install` but I see that a nonexistent `BINDIR` is causing the failure in the `make` stage in your test (even though the installation was successful locally and on Travis). I had to export it separately in `prebuildopts` and `postinstallcmds`, which isn't very clean but I couldn't find another way.

Before this commit I tried to do the customized make install commands in `install_cmd` but that was ignored by EasyBuild. (Although `eb -a -e MakeCp` shows `install_cmd` is a valid option in MakeCp.) So I had to put those under `postinstallcmds`.

Reply:
I forgot to change `%(name)s` to `%(namelower)s` but it still passed the test